pkgs/games/jfsw/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, git
+, which
+, SDL2
+, perl
+, pkg-config
+, wrapGAppsHook
+, gtk3
+}:
+
+stdenv.mkDerivation {
+  pname = "jfsw";
+  version = "stable-20211225";
+
+  src = fetchFromGitHub {
+    owner = "jonof";
+    repo = "jfsw";
+    # version name confuses git
+    rev = "426fc188a9b7fdf140d7b5292c4ecd2ada6f8185";
```

```suggestion
    rev = "refs/tags/${finalAttrs.version}";
```
We can solve this by explicitly indicating the tag as `refs/tags/version`.